examples/bank/bank.go

```diff
@@ -45,44 +42,25 @@ func makeAccountID(id int) []byte {
 
 // Bank stores all the bank related state.
 type Bank struct {
-	kvClient     *client.KV
+	db           *client.DB
 	numAccounts  int
 	numTransfers int32
 }
 
-// Helper function to read bytes received on a Get Call response.
-// We could add this to the API
-func readBytes(call client.Call) []byte {
-	if gr := call.Reply.(*proto.GetResponse); gr.Value != nil {
-		return gr.Value.Bytes
-	}
-	return nil
-}
-
-// Helper function to read an int64 received on a Get Call response
-func readInt64(call client.Call) (int64, error) {
-	if resp := readBytes(call); resp != nil {
-		return strconv.ParseInt(string(resp), 10, 64)
-	}
-	return 0, nil
-}
-
 // Read the balances in all the accounts and return them.
-func (bank *Bank) readAllAccountsWithScan() []int64 {
+func (bank *Bank) readAllAccounts() []int64 {
```

Comment:
Unless you're using the individual balances (which it looks like you aren't), I think this would be better as `sumAllAccounts` which returns a single `int64`.